app/helpers/layout_helper.rb

```diff
@@ -98,6 +98,29 @@ def javascript(*args)
     content_for(:javascripts) { javascript_include_tag(*args) }
   end
 
+  def javascript_include_tag(*params, **kwargs)
+    # Workaround for overriding javascript load with webpack_asset_paths, should be removed when webpack_asset_paths is removed
+    if kwargs[:source] == "webpack_asset_paths"
+      kwargs[:webpacked]
```

Comment:
There's no `super` call here, so it does nothing. Is that intentional, or did you mean to unconditionally call `super(...)`?

Reply:
we dont want to call super cause it returns a script tag, 
and the webpacked key contains for example: `webpacked=>"<script>\n//<![CDATA[\nwindow.tfm.tools.loadPluginModule('/webpack/foreman_remote_execution','foreman_remote_execution','./index');\n//]]>\n</script>"`
which we declare  in: ` webpacked: webpacked_plugins_js_for(plugin_name.to_sym)`